packages/compile-solidity/src/index.js

```diff
@@ -129,6 +129,12 @@ const Compile = {
     options = Config.default().merge(options);
     options = normalizeOptions(options);
 
+    const supplier = new CompilerSupplier({
+      events: options.events,
+      solcConfig: options.compilers.solc
+    });
+    const { solc } = await supplier.load();
+
```

Comment:
would be nice to log some metadata about the loaded solc

Reply:
Pushed a change to add this in `compile-solidity:compilerSupplier`, rather than here directly